web/src/lib/stores/activity.store.ts

```diff
@@ -0,0 +1,12 @@
+import { writable } from 'svelte/store';
+
+export const numberOfComments = writable<number | undefined>(undefined);
+
+export const setNumberOfComments = (number: number) => {
+  numberOfComments.set(number);
+};
+
+export const updateNumberOfComments = (addOrRemove: boolean) => {
```

Comment:
What does it mean if addOrRemove is true? Without looking at the code this isn't clear.
I would prefer to have something like e.g. `addOrRemove: 'add' | 'remove'` or even just let them specify the delta `delta: 1 | -1`.